packages/govuk-frontend/src/govuk/helpers/typography.unit.test.js

```diff
@@ -373,14 +373,7 @@ describe('@mixin govuk-typography-responsive', () => {
       })
 
       await expect(results).resolves.toMatchObject({
-        css: expect.stringContaining(outdent`
-          @supports (font-variant-numeric: tabular-nums) {
-            .foo {
-              font-feature-settings: normal;
-              font-variant-numeric: tabular-nums;
-            }
-          }
-        `)
+        css: expect.stringContaining('font-variant-numeric: tabular-nums;')
```

Comment:
Thanks for tidying this up. Could we also add some tests just for `govuk-typography-tabular-numbers` that test it plain and with the `$important` param?